Description: Include config option within qmake build to support "noopt"
  New behaviour is that "qmake" will make a "release" build a before, while
  "qmake -config noopt" will make a "debug" build with no optimisation.
Author: Stuart Prescott <stuart+debian@nanonanonano.net>

--- a/app/app.pro
+++ b/app/app.pro
@@ -80,6 +80,12 @@
 
 ### Output
 
+# Don't use compiler optimisations if required
+noopt {
+	CONFIG -= release
+	CONFIG += debug
+}
+
 TARGET = qtikz
 target.path = $${BINDIR}
 INSTALLS += target
